lib/src/migrators/module.dart

```diff
@@ -1219,17 +1248,16 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
 
   /// Finds the namespace for the stylesheet containing [declaration], adding a
   /// new `@use` rule if necessary.
-  String _namespaceForDeclaration(MemberDeclaration declaration) {
-    if (declaration == null) return null;
-
+  String? _namespaceForDeclaration(MemberDeclaration declaration) {
     var url = declaration.sourceUrl;
     if (url == currentUrl) return null;
 
     // If we can load [declaration] from a library entrypoint URL, do so. Choose
     // the shortest one if there are multiple options.
     var libraryUrls = references.libraries[declaration];
-    if (libraryUrls != null) {
-      url = minBy(libraryUrls, (url) => url.pathSegments.length);
+    if (libraryUrls != null && libraryUrls.isNotEmpty) {
+      Uri? minUrl = minBy(libraryUrls, (url) => url.pathSegments.length);
```

Comment:
Generally I think it's cleaner to try to keep all type stuff on the RHS if possible, so if inference isn't working here I'd write:

```suggestion
      var minUrl = minBy<Uri>(libraryUrls, (url) => url.pathSegments.length);
```

Same below.

Reply:
`minBy` has two type parameters, so is this still cleaner as `minBy<Uri, int>`?

Reply:
Yeah, I think so.